### PR TITLE
[CPU] Added vector size tiles alignment for quantization buffers

### PR DESCRIPTION
**Pull request by dmitry-gorokhov**
Issues: 46427

### CODE REVIEW

**Comment by dmitry-gorokhov** [comment]
@mandrono could you please review?

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_quantize_node.cpp [hunk 1]
```diff
@@ -534,6 +534,23 @@ void MKLDNNQuantizeNode::execute(mkldnn::stream strm) {
 void MKLDNNQuantizeNode::appendPostOps(mkldnn::post_ops& ops) {
     if (!isPostOpDataInitialized) {
         isPostOpDataInitialized = true;
+
+        // MKLDNN quantization_injectors assumes that quantization data memory is always aligned on 16 (by vector length)
+        // Otherwise it can lead to buffer over-read and performance penalties due to denormals.
+        size_t bufferAlignment = 16;
```
Comment (mandrono) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_quantize_node.cpp:540]:
```suggestion
        const size_t bufferAlignment = 16;
```

Comment (dmitry-gorokhov) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_quantize_node.cpp:540]: Why we need this change?

Comment (mandrono) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_quantize_node.cpp:540]: just minor comment
looks more correctly use const for unchanged variable

**Review by mandrono — APPROVED** [review]

**Pull request merged**